### ISSUE TITLE
Implement Chunked Binary Messages in Octopus

### ISSUE THREAD

**Issue by ascibisz** [simularium/simularium-planning#157]
## Use Case
As a developer, I want octopus to send binary frame data in a chunked format, not exceeding a maximum message size, matching the functionality in the simularium-engine.

## Acceptance Criteria
* Determine a good maximum message size, which can just be confirming what is used in simularium-engine. Make sure this threshold aligns with documentation for the python package we are using in Octopus, as well as the websocket library used by the simularium-viewer
* If a frame exceeds a maximum message size, split it into multiple messages
* Send multiple frames in one message when possible like simularium-engine does, if determined to be worthwhile
* Write tests

## Details
Determine root cause of (this bug)[https://github.com/simularium/simularium-website/issues/421] and how it should be addressed in Octopus before starting this ticket. If there is a race condition caused by batched frame data that can lead to an error state in the viewer, we should consider only splitting frames when a single frame exceeds the size limit, rather than consistently splitting frames as the simularium-engine currently does.
